ui/src/app/pages/admin/admin-workspace.tsx

```diff
@@ -216,19 +217,23 @@ interface State {
   workspaceDetails?: WorkspaceAdminView;
   cloudStorageTraffic?: CloudStorageTraffic;
   loadingData?: boolean;
+  loadingLockUnlockStatus: boolean;
   runtimeToDelete?: ListRuntimeResponse;
   confirmDeleteRuntime?: boolean;
   dataLoadError?: Response;
   files?: Array<FileDetail>;
+  showLockWorkspaceModal: boolean;
 }
 
-class AdminWorkspaceImpl extends React.Component<Props, State> {
+export class AdminWorkspaceImpl extends React.Component<Props, State> {
   constructor(props) {
     super(props);
 
     this.state = {
       workspaceDetails: {},
       cloudStorageTraffic: null,
+      loadingLockUnlockStatus: false,
```

Comment:
name is confusing to me: How about just `loadingAdminLockedStatus`?

Reply:
Done